src/tasks/MonoTargetsTasks/ILStrip/ILStrip.cs

```diff
@@ -12,45 +12,95 @@
 using CilStrip.Mono.Cecil.Binary;
 using CilStrip.Mono.Cecil.Cil;
 using CilStrip.Mono.Cecil.Metadata;
+using System.Reflection.Metadata;
```

Comment:
Are we still depending on `Cecil` library?

Reply:
Yes, for trimming away the whole assemblies.